pkg/virt-operator/resource/apply/rbac_test.go

```diff
@@ -285,6 +286,7 @@ var _ = Describe("RBAC test", func() {
 			required := newEmptyResource(resourceType)
 
 			assignRulesToRoles(newFakePolicyRules("policy1"), existing, required)
+			getRbacMetaObject(required).OwnerReferences = []metav1.OwnerReference{}
```

Comment:
hmm I assumed the unit tests failures were CDI-related, interesting, what is happening here?

Reply:
I don't think they are CDI related, it is likely some k8s dependency that is now returning an empty list of ownerRefs vs a nil.